openmdao/core/problem.py

```diff
@@ -343,7 +361,7 @@ def cleanup(self):
         self.driver.cleanup()
 
     def setup(self, vector_class=DefaultVector, check=True, logger=None, mode='rev',
-              force_alloc_complex=False):
+              force_alloc_complex=False, print_citations=False):
```

Comment:
Why are we adding another argument to setup here when you've already made this accessible as an 'openmdao' command?  In the unlikely event that a user wants to do this programmatically, they can just do the necessary import and call find_citations(prob).  If we're worried about user convenience, it seems like the 'openmdao cite' command is the most convenient way to access this functionality.

Reply:
I asked for this change so that for the non-command line way, the user doesn't have to do the extra import. Not a big deal for me either way though

Reply:
using the `openmdao` command is pretty easy, though it doesn't yet work if you have a file with multiple problems. Citations are something people are lazy about, so asking them to remember an import is likely to result in less people doing it.

I guess we could push the `openmdao` method and not offer two ways to do it ... 
but I don't really see a huge advantage to the direct function call vs the setup arg, and the setup arg requires one less step for the user

Reply:
finding the citations is only one of many post-setup functions that are best called by the user from the command line.  Do we want to add all of these as args to the Problem setup command to handle the uncommon use case where a user wants to call them programmatically?  The issue of the openmdao command not working with scripts having multiple problems could be fixed by allowing problems to have a 'name' attribute so that problems could be specified by name from the command line.

Reply:
Could just make it a method on problem too, instead of an argument.